runtimes.json

```diff
@@ -1,11 +1,13 @@
 {
     "aliases": {
         "latest": "0.3.7-R2-2-1",
-        "openmp": "openmp-rc2",
+        "openmp": "openmp-120",
         "0.3.DL": "0.3.DL-R1",
         "0.3.7": "0.3.7-R2-2-1",
         "0.3z": "0.3z-R4",
-        "UGMP": "UG-MP"
+        "UGMP": "UG-MP",
+        "openmp-rc2": "openmp-rc2",
```

Comment:
Remove these aliases as they point to the same version and are not needed, aliases are for refering to changing versionings like pointing to the latest of a specific set of packages